File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    op = {'-': -1, '+': 1}
    for me in quiz:
        me = me.split()
        ans = 'O' if int(me[0]) + op[me[1]] * int(me[2]) == int(me[4]) else 'X'
        answer.append(ans)
    return answer

File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        for (int i = 0; i < quiz.length; i++) {
            String[] me = quiz[i].split(" ");
            int a = Integer.parseInt(me[0]);
            int b = Integer.parseInt(me[2]);
            int c = Integer.parseInt(me[4]);
            int op = me[1].equals("-") ? -1 : 1;
            answer[i] = a + op * b == c ? "O" : "X";
        }
        return answer;
    }
}